modules/core/src/test/resources/catalogs/landsat-stac/landsat-8-l1/catalog.json

```diff
@@ -135,27 +136,19 @@
   "links" : [
     {
       "href" : "../../catalog.json",
-      "rel" : "root",
-      "label:assets" : [
-      ]
+      "rel" : "root"
     },
     {
       "href" : "../../catalog.json",
-      "rel" : "parent",
-      "label:assets" : [
-      ]
+      "rel" : "parent"
     },
     {
       "href" : "./catalog.json",
-      "rel" : "self",
-      "label:assets" : [
-      ]
+      "rel" : "self"
     },
     {
       "href" : "./2014-153/LC81530252014153LGN00.json",
-      "rel" : "item",
-      "label:assets" : [
-      ]
+      "rel" : "item"
```

Comment:
doing all this json editing was a nuisance but we had it coming :disappointed: it's what i meant when i talked about the maintenance burden on static catalogs

Reply:
I don't think it replaces the need for having some "hand-rolled STAC" to test against - but another option would be to use [`circe-golden`](https://github.com/circe/circe-golden) which would use the arbitrary  instances to generate some static JSON that is then used to check that the encoders/decoders work with that JSON in future runs.

We can probably hold off until STAC reaches 1.0, but once it does we should use this to ensure we maintain backwards compatibility.

Reply:
#88